samples/qa-a11y/src/views/ScrollerWithBodyTextFocus.js

```diff
@@ -0,0 +1,132 @@
+			Mauris blandit sollicitudin mattis. Fusce commodo arcu vitae risus consectetur<br />
+			sollicitudin. Aliquam eget posuere orci. Cras pellentesque lobortis sapien non lacinia.<br />
+		</>,
+		<>
+			Lorem ipsum dolor sit amet, consectetur adipiscing elit.<br />
+			Aenean id blandit nunc. Donec lacinia nisi vitae mi dictum, eget pulvinar nunc tincidunt.<br />
+			Integer vehicula tempus rutrum. Sed efficitur neque in arcu dignissim cursus.<br />
+		</>,
+		<>
+			Mauris blandit sollicitudin mattis. Fusce commodo arcu vitae risus consectetur<br />
+			sollicitudin. Aliquam eget posuere orci. Cras pellentesque lobortis sapien non lacinia.<br />
+		</>,
+		<>
+			Lorem ipsum dolor sit amet, consectetur adipiscing elit.<br />
+			Aenean id blandit nunc. Donec lacinia nisi vitae mi dictum, eget pulvinar nunc tincidunt.<br />
+			Integer vehicula tempus rutrum. Sed efficitur neque in arcu dignissim cursus.<br />
+		</>,
+		<>
+			Mauris blandit sollicitudin mattis. Fusce commodo arcu vitae risus consectetur<br />
+			sollicitudin. Aliquam eget posuere orci. Cras pellentesque lobortis sapien non lacinia.<br />
+		</>,
+		<>
+			Lorem ipsum dolor sit amet, consectetur adipiscing elit.<br />
+			Aenean id blandit nunc. Donec lacinia nisi vitae mi dictum, eget pulvinar nunc tincidunt.<br />
+			Integer vehicula tempus rutrum. Sed efficitur neque in arcu dignissim cursus.<br />
+		</>,
+		<>
+			Mauris blandit sollicitudin mattis. Fusce commodo arcu vitae risus consectetur<br />
+			sollicitudin. Aliquam eget posuere orci. Cras pellentesque lobortis sapien non lacinia.<br />
+		</>,
+		<>
+			Lorem ipsum dolor sit amet, consectetur adipiscing elit.<br />
+			Aenean id blandit nunc. Donec lacinia nisi vitae mi dictum, eget pulvinar nunc tincidunt.<br />
+			Integer vehicula tempus rutrum. Sed efficitur neque in arcu dignissim cursus.<br />
+		</>,
+		<>
+			Mauris blandit sollicitudin mattis. Fusce commodo arcu vitae risus consectetur<br />
+			sollicitudin. Aliquam eget posuere orci. Cras pellentesque lobortis sapien non lacinia.<br />
+		</>,
+		<>
+			Lorem ipsum dolor sit amet, consectetur adipiscing elit.<br />
+			Aenean id blandit nunc. Donec lacinia nisi vitae mi dictum, eget pulvinar nunc tincidunt.<br />
+			Integer vehicula tempus rutrum. Sed efficitur neque in arcu dignissim cursus.<br />
+		</>,
+		<>
+			Mauris blandit sollicitudin mattis. Fusce commodo arcu vitae risus consectetur<br />
+			sollicitudin. Aliquam eget posuere orci. Cras pellentesque lobortis sapien non lacinia.<br />
+		</>
+	];
+
+	const Container = SpotlightContainerDecorator('div');
+	const SpottableDiv = Spottable('div');
+
+	return (
+		<Layout orientation="vertical">
+			<Cell shrink>
+				<CheckboxItem
+					alt="Toggle"
+					selected={audioGuidance}
+					disabled={toggleDisabled}
+					onToggle={onToggle}
+				>
+					Audio guidance
+				</CheckboxItem>
+			</Cell>
+			<Cell
+				component={Scroller}
+				focusableScrollbar={audioGuidance ? true : "byEnter"}
```

Comment:
I think it make sense to make focusableScrollbar `false` when audioGuidance is `on`.
```suggestion
				focusableScrollbar={audioGuidance ? false : "byEnter"}
```

Reply:
I accepted your suggestion.